pages/settings/admin/playback.vue

```diff
@@ -0,0 +1,157 @@
+        {{ $t('settings.playback.video.controlsDescription') }}
+      </v-card-subtitle>
+      <v-divider />
+
+      <div class="pa-5">
+        <v-text-field
+          v-model="backwardSeek"
+          :value="backwardSeek"
+          :rules="[validateInput]"
+          min="0"
+          :hint="$t('settings.playback.video.default')"
+          prepend-icon="mdi-skip-backward"
+          :suffix="$t('settings.playback.video.seconds')"
+          type="number"
+          :label="$t('settings.playback.video.backwardSeek')"
+          class="input-text"
+        />
+        <v-text-field
+          v-model="forwardSeek"
+          :value="forwardSeek"
+          :rules="[validateInput]"
+          min="0"
+          :hint="$t('settings.playback.video.default')"
+          prepend-icon="mdi-skip-forward"
+          :suffix="$t('settings.playback.video.seconds')"
+          type="number"
+          :label="$t('settings.playback.video.forwardSeek')"
+          class="input-text"
+        />
+      </div>
+
+      <v-row justify="center">
+        <v-card-actions>
+          <v-icon>mdi-music</v-icon>
+        </v-card-actions>
+        <v-card-title>
+          <h1 class="text-overline title">
+            {{ $t('settings.playback.music.music') }}
+          </h1>
+        </v-card-title>
+      </v-row>
+
+      <v-card-title>
+        {{ $t('settings.playback.music.crossfade') }}
+      </v-card-title>
+      <v-card-subtitle>
+        {{ $t('settings.playback.music.crossfadeDescription') }}
+      </v-card-subtitle>
+      <v-divider />
+
+      <v-row justify="space-between" align="center" class="pa-5">
+        <v-card-actions>
+          <v-fade-transition>
+            <v-slider
+              v-show="crossfade"
+              v-model="crossfadeTime"
+              min="1"
+              max="12"
+              thumb-label
+              class="slider"
+            >
+              <template #append>
+                <span class="mt-1">
+                  {{ crossfadeTime + 's' }}
+                </span>
+              </template>
+            </v-slider>
+          </v-fade-transition>
+        </v-card-actions>
+        <v-card-actions>
+          <v-switch v-model="crossfade" />
+        </v-card-actions>
+      </v-row>
+
+      <v-card-title>
+        {{ $t('settings.playback.music.queueSync') }}
+      </v-card-title>
+      <v-card-subtitle>
+        {{ $t('settings.playback.music.queueSyncDescription') }}
+      </v-card-subtitle>
+      <v-divider />
+
+      <v-row justify="end" align="end" class="pa-5">
+        <v-card-actions>
+          <v-switch v-model="queueSync" />
+        </v-card-actions>
+      </v-row>
+    </template>
+  </settings-page>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import { mapActions } from 'vuex';
+
+export default Vue.extend({
+  data() {
+    return {
+      crossfade: false,
+      crossfadeTime: 0,
+      queueSync: false,
+      backwardSeek: 20,
+      forwardSeek: 20
```

Comment:
We shouldn't set these defaults here, we should request from server and then store in data, until then it should be undefined